ci-scripts/e2e-cypress.sh

```diff
@@ -93,9 +93,9 @@ else
     echo '-----'
     echo "Running Cypress end to end tests"
 
-    if [ "${GITHUB_EVENT_NAME}" == "pull_request" ]; then
-        yarn e2e:run:ci:core"${SUITE}"
-    else
+#    if [ "${GITHUB_EVENT_NAME}" == "pull_request" ]; then
+#        yarn e2e:run:ci:core"${SUITE}"
+#    else
```

Comment:
This looks like a temp change that should be reverted.

Reply:
Correct. Thanks for pointing it out.